src/lib/storage/storage_manager.hpp

```diff
@@ -70,6 +103,37 @@ class StorageManager : public Noncopyable {
   tbb::concurrent_unordered_map<std::string, std::shared_ptr<Table>> _tables{INITIAL_MAP_SIZE};
   tbb::concurrent_unordered_map<std::string, std::shared_ptr<LQPView>> _views{INITIAL_MAP_SIZE};
   tbb::concurrent_unordered_map<std::string, std::shared_ptr<PreparedPlan>> _prepared_plans{INITIAL_MAP_SIZE};
+
+ private:
+  static const uint32_t CHUNK_COUNT = 50;
```

Comment:
Private Members should start with a leading underscore.
Same for all the other constants.

Reply:
💡 TIL